mirgecom/boundary.py

```diff
@@ -71,3 +75,48 @@ def boundary_pair(
     ):
         dir_soln = discr.project("vol", btag, q)
         return TracePair(btag, interior=dir_soln, exterior=dir_soln)
+
+
+class AdiabaticSlipBoundary:
+    """
+    Adiabatic slipwall boundary for inviscid flows
```

Comment:
"slipwall" doesn't seem to be a dictionary word.

Reply:
How dare they?  Maybe we should write the editor.